specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
To remedy this discussion and provide us a means by which to reason about ballots under audit from multiple perspectives, I have renamed the original table mostly per @nealmcb's recommendation (relating counties to sequences of ballot manifest info) and I have added another table to continue to capture the old relationship to contests.